lib/cirro_io_v2/resources/notification_channel.rb

```diff
@@ -0,0 +1,13 @@
+module CirroIOV2
+  module Resources
+    class NotificationChannel < Base
```

Comment:
@garciaf Here, we need to change NotificationChannel to NotificationTopic. See api doc https://cirroapiv2.docs.apiary.io/#reference/0/notification-topics/create-a-notification-topic